src/ExposureHistory/History/DateInfoHeader.tsx

```diff
@@ -17,6 +21,8 @@ const DateInfoHeader: FunctionComponent<DateInfoHeaderProps> = ({
   lastDetectionDate,
 }) => {
   const { t } = useTranslation()
+  const { checkForNewExposures } = useExposureContext()
+  const [refreshing, setRefreshing] = useState(false)
```

Comment:
Given that checking exposures is on the order of milliseconds, what do you think about not showing a loading state?  I assume the loading indicator would always just flash, so theres no really need to give visual feedback to the user, and it would be a bit less code and a bit smoother of a user experience to not have the loading state.

Reply:
hm, I might push to leave this in place; if the user has a poor network connection and there are a handful of key archives to pull down, it could conceivably take a few seconds. You're quite right, more often than not it will be nearly instantaneous. We can add an animation to improver the user experience a tad: 
![Sep-27-2020 11-44-03](https://user-images.githubusercontent.com/2637355/94369410-824c8480-00b7-11eb-8b56-f381f05c1bed.gif)

Reply:
Ah, that makes total sense to me. If its possible to take more than an instant we definitely want the loading state.